cognite/client/data_classes/geospatial.py

```diff
@@ -89,14 +95,16 @@ class Feature(CogniteResource):
 
     PRE_DEFINED_SNAKE_CASE_NAMES = {utils._auxiliary.to_snake_case(key) for key in RESERVED_PROPERTIES}
 
-    def __init__(self, external_id: str = None, cognite_client=None, **properties):
+    def __init__(self, external_id: str = None, cognite_client: Any = None, **properties: Any):
         self.external_id = external_id
         for key in properties:
             setattr(self, key, properties[key])
-        self._cognite_client = cognite_client
+        self._cognite_client = cast(Any, cognite_client)
```

Comment:
isn't this unsafe op?

Reply:
Yeah, it's not ideal. It's just a hack to get around the circular dep. But I think I actually have a better way of solving this.